package.json

```
@@ -1,45 +1,46 @@
 {
 	"private": true,
 	"scripts": {
-		"lint": "npm run lint:css && npm run lint:js",
+		"lint": "run-p lint:*",
 		"lint:js": "xo",
 		"lint:css": "stylelint source/**/*.css",
-		"lint-fix": "npm run lint:css -- --fix && npm run lint:js -- --fix",
-		"test": "npm run lint && npm run test:js && npm run build",
+		"lint-fix": "run-p 'lint:* -- --fix'",
+		"test": "run-s lint:* test:* build",
 		"test:js": "ava",
 		"build": "webpack --mode=production",
 		"watch": "webpack --mode=development --watch",
 		"prerelease:version": "VERSION=$(utc-version); echo $VERSION; dot-json distribution/manifest.json version $VERSION",
-		"prerelease:source-url": "echo https://github.com/sindresorhus/notifier-for-github/tree/\"${TRAVIS_COMMIT:-$VERSION}\" > distribution/SOURCE_URL",
+		"prerelease:source-url": "if [[ $TRAVIS_REPO_SLUG ]]; then echo https://github.com/$TRAVIS_REPO_SLUG/tree/\"${TRAVIS_TAG:-$TRAVIS_COMMIT}\" > distribution/SOURCE_URL; fi",
 		"release": "npm-run-all build prerelease:* release:*",
 		"release:cws": "webstore upload --source=distribution --auto-publish",
 		"release:amo": "web-ext-submit --source-dir distribution"
 	},
 	"dependencies": {
 		"delay": "^4.3.0",
-		"webext-options-sync": "^0.16.0",
+		"webext-options-sync": "^0.21.2",
```

You can try the latest, while you're at it

```suggestion
		"webext-options-sync": "^1.0.0-4",
```

```sh
npm i webext-options-sync@next
```

---

Trying to stay on stable for now, planning on a rewrite once everything is available.

Love the new additions on `webext-options-sync` though. 😄